R/bayesianmetaanalysiscommon.R

```diff
@@ -19,6 +19,8 @@
 
 .BayesianMetaAnalysisCommon <- function(jaspResults, dataset, ready, options) {
   
+  .setSeedJASP(options)
```

Comment:
it might be better to put it right before the fitting function, in case that the order of flow changes / there are other rng elements